service-catalog/docs-rs/REAMDE.md

```diff
@@ -0,0 +1,15 @@
+# Docs.rs
+
+Docs.rs is a service that hosts documentation of crates.
+
+The docs.rs codebase is available at [rust-lang/docs.rs] and
+it's maintained by the [docs.rs team](https://www.rust-lang.org/governance/teams/dev-tools#team-docs-rs).
+
+The codebase is deployed via the [terraform/docs-rs] module.
```

Comment:
@MarcoIeni to my knowledge this is not fully true. 

Most of the deployment is done manually on the EC2 machine, 

parts of the infra are managed by terraform (cloudfront, s3 for example), 

but I think some things are just the "new" WIP infra, ( RDS / webserver), and not what we use in prod